src/RecoveryServices/RecoveryServices.SiteRecovery.Test/ScenarioTests/B2A/AsrB2ATests.ps1

```diff
@@ -814,30 +814,51 @@ function Test-CreateRPIWithProximityPlacementGroup
     $pcm = Get-ASRProtectionContainerMapping -ProtectionContainer $pc
     $policy = Get-AzRecoveryServicesAsrPolicy -Name $PolicyName
     $VM= Get-AsrProtectableItem -ProtectionContainer $pc -FriendlyName $VMName
-    $ppg = "/subscriptions/b364ed8d-4279-4bf8-8fd1-56f8fa0ae05c/resourceGroups/Arpita-air/providers/Microsoft.Compute/proximityPlacementGroups/h2atestppgenable"
-    $EnableDRjob = New-AsrReplicationProtectedItem -ProtectableItem $VM -Name $VM.Name -ProtectionContainerMapping $pcm -RecoveryAzureStorageAccountId $StorageAccountID -OSDiskName $VMName -OS Windows -RecoveryResourceGroupId $RecoveryResourceGroupId -RecoveryProximityPlacementGroupId $ppg -UseManagedDisk true
+    $ppg = "/subscriptions/b364ed8d-4279-4bf8-8fd1-56f8fa0ae05c/resourceGroups/prakccyrg/providers/Microsoft.Compute/proximityPlacementGroups/h2appgenable"    
+    $avset="/subscriptions/b364ed8d-4279-4bf8-8fd1-56f8fa0ae05c/resourceGroups/prakccyrg/providers/Microsoft.Compute/availabilitySets/h2aavsetenable"
+    $size = "Standard_B1s"
+    $sqlLicenseType = "AHUB"
+    $vmTag = New-Object "System.Collections.Generic.Dictionary``2[System.String,System.String]"
+    $vmTag.Add("VmTag1","powershellVm")
+    $diskTag = New-Object "System.Collections.Generic.Dictionary``2[System.String,System.String]"
+    $diskTag.Add("DiskTag1","powershellDisk")
+    $nicTag = New-Object "System.Collections.Generic.Dictionary``2[System.String,System.String]"
+    $nicTag.Add("NicTag1","powershellNic")
+    $EnableDRjob = New-AsrReplicationProtectedItem -ProtectableItem $VM -Name $VM.Name -ProtectionContainerMapping $pcm -RecoveryAzureStorageAccountId $StorageAccountID -OSDiskName $VMName -OS Windows -RecoveryResourceGroupId $RecoveryResourceGroupId -RecoveryProximityPlacementGroupId $ppg -UseManagedDisk true -RecoveryAvailabilitySetId $avset -Size $size -SqlServerLicenseType $sqlLicenseType -RecoveryVmTag $vmTag -RecoveryNicTag $nicTag -DiskTag $diskTag -RecoveryAzureNetworkId $AzureVmNetworkId
```

Comment:
>AsrReplicationProtectedItem [](start = 23, length = 27)

In future we should wait for enable and IR to finish and check all these are coming fine, for now u can check in armclient

Reply:
For now checked from portal. Will do it next release.

---
In reply to: [601630510](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601630510) [](ancestors = 601630510)